packages/lib/response/service.ts

```diff
@@ -95,6 +95,12 @@ export const responseSelection = {
       },
       isResolved: true,
       isEdited: true,
+      response: {
```

Comment:
why is that needed? we already fetch the response id and surveyId

Reply:
I added this earlier as I got some errors in the types inference. To fix this, I've extended the select value for the `getResponseNote` function, ensuring that other data-fetching calls remain the same.